projects/ebv-analyzer-app/src/app/ebv-selector/ebv-selector.component.ts

```diff
@@ -612,3 +566,15 @@ function guessDataRange(colorizer: Colorizer): {min: number; max: number} {
 
     return {min, max};
 }
+
+// TODO: use method from core
```

Comment:
why don't you use it? I guess it is already there

Reply:
the method is not exported and I don't know how to do it.

Reply:
You have to add the file where it is in into the `public-api.ts`.
I guess it is just a model, so you can sort it in there.
Please keep the sorting for a better overview.

Reply:
the public-api.ts only exports Services, Components, Directives and Models but not plain functions. That's why I don't know how to use those in another project.

Reply:
You can either put it under models or misc.
For instance, `form.validators.ts` has `isValidUuid` which you can import since it is `export`ed.
Or `rgbaColorFromDict` from `color.ts`.

Reply:
how do I import it then from a different project? both functions you mention are only used in the core

Reply:
You import them the same way you import components, directives and so on.

`import {rgbaColorFromDict} from 'wave-core';`

Reply:
when I first tried that it said that it doesn't know that component but it works
7697d8327a582eb6fec3147830fff733f91de6ab